app/setup.py

```diff
@@ -44,8 +44,8 @@
 
 CSP_POLICY = {
     'default-src': ["'self'", ],
-    'script-src': ["'self'", 'strict-dynamic', ],
-    'img-src': ["'self'", "data:"]
+    'script-src': ["'self'", 'www.google-analytics.com', ],
```

Comment:
I think we can force this to only allow https too

Reply:
Fair enough, I can't think of a good reason why not.